doc/dev/cephfs-mirroring.rst

```diff
@@ -441,53 +455,66 @@ E.g., adding a regular file for synchronization would result in failed status::
     }
   }
 
-This allows a user to add a non-existent directory for synchronization. The mirror daemon
-would mark the directory as failed and retry (less frequently). When the directory comes
-to existence, the mirror daemons would unmark the failed state upon successful snapshot
-synchronization.
+This allows a user to add a non-existent directory for synchronization. The
+mirror daemon marks the directory as failed and retries (less frequently).
+When the directory comes into existence, the mirror daemons notice the
+successful snapshot synchronization and unmark the failed state. 
+
+When mirroring is disabled, the ``fs mirror status`` command for the file
+system will not show up in command help.
 
-When mirroring is disabled, the respective `fs mirror status` command for the file system
-will not show up in command help.
+The mirroring module provides a couple of commands to display directory mapping
+and distribution information. To check which mirror daemon a directory has been
+mapped to use, run a command of the following form:
 
-Mirroring module provides a couple of commands to display directory mapping and distribution
-information. To check which mirror daemon a directory has been mapped to use::
+.. prompt:: bash $
+
+   ceph fs snapshot mirror dirmap cephfs /d0/d1/d2
+::
 
-  $ ceph fs snapshot mirror dirmap cephfs /d0/d1/d2
   {
     "instance_id": "404148",
     "last_shuffled": 1601284516.10986,
     "state": "mapped"
   }
 
-.. note:: `instance_id` is the RADOS instance-id associated with a mirror daemon.
+.. note:: ``instance_id`` is the RADOS instance-id associated with a mirror
+   daemon.
 
-Other information such as `state` and `last_shuffled` are interesting when running
-multiple mirror daemons.
+Other information such as ``state`` and ``last_shuffled`` are interesting when
```

Comment:
including.

"interesting"?   I'm not convinced that this sentence adds value, so I would take it out completely.